.github/CONTRIBUTING.md

```diff
@@ -30,6 +30,36 @@ Our kanban board is public and available [here](https://github.com/orgs/vazco/pr
 - Make sure you have added the necessary tests for your changes. Do not worry though, the Codecov bot will report it in the pull request.
 - Make sure your code passes _all_ tests: `npm test`.
 
+### Development
+
+For the best developer experience (DX) when developing locally it is recommended to start `build` and `test` in a `--watch` mode.
```

Comment:
The `a` is redundant in "a --watch mode". Should either be just "in --watch mode" or "in the --watch mode".

In general you could simplify the start of this sentence but it's subjective so I'm okay with this version too. "For the best local developer experience"

Reply:
That's better!